rest/admin_api.go

```diff
@@ -131,6 +136,19 @@ func (h *handler) handleGetDbConfig() error {
 	includeJavascript, _ := h.getOptBoolQuery("include_javascript", true)
 	_ = includeJavascript
 
+	if h.server.bootstrapContext.connection != nil {
+		found, dbConfig, err := h.server.fetchDatabase(h.db.Name)
+		if err != nil {
+			return err
+		}
+
+		h.response.Header().Set("ETag", dbConfig.Version)
+
+		if !found {
+			return base.HTTPErrorf(http.StatusNotFound, "database config not found")
+		}
+	}
```

Comment:
Don't think so. We're not returning the same config that we're fetching here. dbConfig that is being used for version isn't the one that we're returning in the body.

Reply:
Oops yeah. Ok, make sure `GetDatabaseConfig` isn't populating Version then :D

Reply:
Yeah true, I think it is. I've switched to have the writeJSON return just the dbConfig rather than the full DatabaseConfig. Thats what redacted version does so probably fine? If you have any issues with this I can explicitly remove it